frontend/src/Onboarding/MajorScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { withRouter, RouteComponentProps, Link } from "react-router-dom";
+import { TextField } from "@material-ui/core";
+import { GenericQuestionTemplate } from "./GenericQuestionScreen";
+import { NextButton } from "../components/common/NextButton";
+import { Autocomplete } from "@material-ui/lab";
+import { majors } from "../majors";
+import { Major } from "../models/types";
+
+interface State {
+  major?: Major;
+}
+
+class MajorComponent extends React.Component<RouteComponentProps, State> {
+  constructor(props: RouteComponentProps) {
+    super(props);
+
+    this.state = {
+      major: undefined,
+    };
+  }
+
+  onChooseMajor(event: React.SyntheticEvent<{}>, value: any) {
+    const maj = majors.find((m: any) => m.name === value);
+
+    this.setState({ major: maj });
+  }
+
+  renderMajorDropDown() {
+    return (
+      <Autocomplete
+        style={{ width: 300 }}
+        disableListWrap
+        options={majors.map(maj => maj.name)}
+        renderInput={params => (
+          <TextField
+            {...params}
+            variant="outlined"
+            label="Select A Major"
+            fullWidth
+          />
+        )}
+        value={!!this.state.major ? this.state.major.name + " " : ""}
```

Comment:
For some reason this doesn't seem to be working right. I don't see the major.name populated when it's selected from the drop-down

Reply:
Also why the `!!` ?

Reply:
This is really weird. It works fine for me when I test it locally using `yarn start`, but it doesn't work on zeit.